Documentation/SpiderBasic/English/Geolocation.txt

```diff
@@ -24,20 +24,20 @@
   Starts geolocation data monitoring.
 
 @OptionalParameter "Timeout"
-  The maximum timeout (in millisecond) to have valid geolocation data. If no data can be updated
-  during this period, all geolocation functions will returns the error value. This can be useful
-  where up-to-date geolocation information are mandatory (ex: for a car navigation application).
+  The maximum timeout (in milliseconds) to have valid geolocation data. If no data can be updated
+  during this period, all geolocation functions will return the error value. This can be useful
+  where up-to-date geolocation information is mandatory (e.g. for a car navigation application).
   If not specified, no timeout is set.
 
 @OptionalParameter "HighAccuracy"
-  If sets to @#True, enable the use of real GPS if available, and fallback to WiFi or other less
-  accurate methods if not found. Default value is @#False, which only use WiFi or less accurate methods for
+  If set to @#True, enable the use of real GPS if available, and fallback to Wi-Fi or other less
+  accurate methods if not found. Default value is @#False, which only use Wi-Fi or less accurate methods for
```

Comment:
There is an inconsitency between the `Wi-Fi` here and the `wi-fi` above at [line 11](https://github.com/fantaisie-software/purebasic/pull/294/files#diff-78df203200603ec2b26d4945f05e25c4ce8b1629ad7ce933468392c1e2dd5a80R11) in regard to upper/loer case. The diff show that error was already present before your fix. However I this you should fix it as well.